task/06-conditions-n-loops-tasks.js

```diff
@@ -81,11 +105,17 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,1,1   =>  false
  *   10,10,10 =>  true
  */
-function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+function isTriangle(a, b, c) {
+    if ((a + b <= c) || (a + c <= b) || (b + c <= a) || (a <= 0) || (b <= 0) || (c <= 0)) {
+        return false;
+    }
+    else {
+        return true;
+    }
```

Comment:
We can simplify this logic by using just one 'return'
Please, see https://javascript.info/logical-operators